### PR TITLE
show only active content sections

### PR DESCRIPTION
**Pull request by Rizziepit**

### CODE REVIEW

**Comment by Rizziepit** [comment]
@miltontony ready for review

File: springboard_iogt/utils.py [hunk 1]
```diff
@@ -50,12 +53,17 @@ def set_indexes(self, s_obj):
...
+        return (slug in cls.DATA and
+                any(index for index in indexes if slug in index))
 
     @classmethod
     def all(cls):
-        return [cls(slug) for slug in cls.SLUGS]
+        return [cls(slug) for slug in cls.DATA.keys()]
+
+    @classmethod
+    def active(cls, indexes):
```
Comment (miltontony) [inline springboard_iogt/utils.py]: I think this more closely represents `known` sections rather than `active`

**Comment by miltontony** [comment]
:+1: aside from 1 potential rename

File: springboard_iogt/utils.py (hunk 1 above)
Comment (Rizziepit) [inline springboard_iogt/utils.py]: Renamed

**Pull request merged**